File: Livro - Módulo Java/3 - Estrutura de Decisão/Exercicio.06.java
import java.util.Scanner;

public class Exercicio6 {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);

        // Leia um número e, se ele for positivo, imprima seu inverso;
        // caso contrário, imprima seu quadrado inverso.

        System.out.print("Digite um número: ");
        double numero = scanner.nextDouble();

        if (numero > 0) {
            double inverso = -1 * numero;
            System.out.println("O inverso do número é: " + inverso);
        } else {
            double inverso2 = -1 * numero;
            double quadrado = Math.pow(inverso2, 2);
            System.out.println("O quadrado do inverso é: " + quadrado);
        }
    }
}

File: Livro - Módulo Python/3 - Estrutura de Decisão/Exercicio.06.py
#6. Leia um número e, se ele for positivo, imprima seu inverso; 
# caso contrário imprima seu quadrado inverso. Inverso: 1/número.

numero = float(input("Digite um número: "))

if numero > 0:
    inverso = -1 * numero
    print("O inverso do número é:", inverso)
else:
    inverso2 = (-1 * numero)
    quadrado =inverso2 ** 2
    print("O quadrado inverso do número é:",quadrado)